plotly/figure_factory/_bullet.py

```diff
@@ -0,0 +1,342 @@
+    :param (int) marker_size: sets the size of the markers in the chart.
+    :param (str | int) marker_symbol: the symbol of the markers in the chart.
+        Default='diamond-tall'
+    :param (list) range_colors: a list of two colors between which all
+        the rectangles for the range are drawn. These rectangles are meant to
+        be qualitative indicators against which the marker and measure bars
+        are compared.
+        Default=['rgb(198, 198, 198)', 'rgb(248, 248, 248)']
+    :param (list) measure_colors: a list of two colors which is used to color
+        the thin quantitative bars in the bullet chart.
+        Default=['rgb(31, 119, 180)', 'rgb(176, 196, 221)']
+    :param (float) horizontal_spacing: see the 'horizontal_spacing' param in
+        plotly.tools.make_subplots. Ranges between 0 and 1.
+    :param (float) vertical_spacing: see the 'vertical_spacing' param in
+        plotly.tools.make_subplots. Ranges between 0 and 1.
+    :param (dict) scatter_options: describes attributes for the scatter trace
+        in each subplot such as name and marker size. Call
+        help(plotly.graph_objs.Scatter) for more information on valid params.
+    :param layout_options: describes attributes for the layout of the figure
+        such as title, height and width. Call help(plotly.graph_objs.Layout)
+        for more information on valid params.
+
+    Example 1: Use a Dictionary
+    ```
+    import plotly
+    import plotly.plotly as py
+    import plotly.figure_factory as ff
+
+    data = [
+      {"e": "Revenue", "d": "US$, in thousands", "c": [150, 225, 300],
+       "b": [220,270], "a": [250]},
+      {"e": "Profit", "d": "%", "c": [20, 25, 30], "b": [21, 23], "a": [26]},
+      {"e": "Order Size", "d":"US$, average","c": [350, 500, 600],
+       "b": [100,320],"a": [550]},
+      {"e": "New Customers", "d": "count", "c": [1400, 2000, 2500],
+       "b": [1000,1650],"a": [2100]},
+      {"e": "Satisfaction", "d": "out of 5","c": [3.5, 4.25, 5],
+       "b": [3.2,4.7], "a": [4.4]}
+    ]
+
+    fig = ff.create_bullet(
+        data, titles='e', subtitles='d', markers='a', measures='b',
+        ranges='c', orientation='h', title='my simple bullet chart'
+    )
+    py.iplot(fig)
+    ```
+
+    Example 2: Use a DataFrame with Custom Colors
+    ```
+    import plotly.plotly as py
+    import plotly.figure_factory as ff
+
+    import pandas as pd
+
+    data = pd.read_json('https://cdn.rawgit.com/plotly/datasets/master/BulletData.json')
+
+    fig = ff.create_bullet(
+        data, titles='title', markers='markers', measures='measures',
+        orientation='v', measure_colors=['rgb(14, 52, 75)', 'rgb(31, 141, 127)'],
+        scatter_options={'marker': {'symbol': 'circle'}}, width=700
+
+    )
+    py.iplot(fig)
+    ```
+    """
+    # validate df
+    if not pd:
+        raise exceptions.ImportError(
+            "'pandas' must be imported for this figure_factory."
```

Comment:
"imported" - it's really installed, not imported, right? The user doesn't need to do a `import pandas` before they call this function do they?
If so, could we correct this typo in any other place that it might appear

Reply:
that's correct, it just needs to be installed on their system